tests/user_restricted/test_user_restricted.py

```diff
@@ -135,16 +151,28 @@ def test_retrieve_patient_with_missing_x_request_header(self, headers_with_token
 
 class TestUserRestrictedSearchPatient:
 
-    # @pytest.mark.smoke_test
     @pytest.mark.nhsd_apim_authorization(AUTH_HEALTHCARE_WORKER)
-    def test_search_patient_happy_path(self, headers_with_token):
+    def test_search_patient_happy_path_for_non_int(self, headers_with_token):
```

Comment:
again, does this need marking so it doesn't run on int (I admit to some ignorance of exactly how pytest works - something i very much need to remedy)

Reply:
See above ... https://github.com/NHSDigital/personal-demographics-service-api/pull/921#discussion_r1331735830